### DISCUSSION TITLE
The only guy at work without copilot

### DISCUSSION THREAD

**Discussion by WMcKibbin** [community/community#10701] [Copilot Conversations]
I'm the lame-O at work without copilot. Getting really frustrated pair programming watching co-pilot finish my sentences.. Help a brother out? :pray:

**Comment by Urethramancer** [comment]
I'm the only one WITH Copilot at work, and I would like the same for my co-workers. They've said they stopped accepting more while planning for the commercial release, or something. I think they should just enable it on paid accounts for now and be done with that side of things. They'd get an awful lot of subscribers at once, I'm sure.
